contracts/SpecificGroupStrategy.sol

```diff
@@ -169,6 +182,14 @@ contract SpecificGroupStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeab
      * strategies.
      */
     function allowStrategy(address group) external onlyOwner {
+        if (
+            (defaultStrategy.getGroupsLength() + specificGroupStrategies.length()) >=
```

Comment:
ditto, regarding check causing unintended revets

Reply:
This check is removed altogether